### DISCUSSION TITLE
Where have all my meta tags gone?

### DISCUSSION THREAD

**Discussion by Elindo586** [vercel/next.js#37782] [Help] [answered]
For example, this page here:

https://github.com/Elindo586/nextprojectb/blob/main/pages/products/flowmeters.js

I have all the meta tags on the Head...   I can see them on the element tab from the console, but I can't see them on the live browser.

They used to show on the live browser, but I deleted some Head tags from the  app.js and now almost all tags are gone from all pages....  the app.js only had a few meta tags and most of the tags where on the individual pages..

**Comment by icyJoseph** [comment]
> I have all the meta tags on the Head... I can see them on the element tab from the console, but I can't see them on the live browser.

What do you mean show live on the browser? Were they actually visible, as in painted, by the browser into the document? Meta tags are not meant to be visible, they are a way to describe the page for other parties, be it bots, crawlers, etc. I am not sure why were they visible to begin with. Since they live in the head of the document they are usually invisible. 

If I make a request to your site I see a bunch of meta tags proper. 

Perhaps this was a CSS issue? `head`, `meta` and `link`, have all display none set, and perhaps I am not sure how, some CSS rules made them have display block? or some other sort of modifier? **That'd be extremely strange though, I don't think it is your case** 

Do you have any screenshots of before and after?

**Comment by Elindo586** [comment, marked as answer]
Apparently it takes time for the meta tags to show up in the live browser when inspecting the console.

Last night I could see the page changes, but inspecting the console I didn't see any meta tags in any of the pages...

This morning, without doing anything extra I went inspect the console again, and sure enough there were all the tags in the head of the document.
